packages/react-router/index.tsx

```diff
@@ -704,6 +704,13 @@ export function createRoutesFromChildren(
       return;
     }
 
+    invariant(
+      element.type === Route,
+      `[${
+        typeof element.type === "string" ? element.type : element.type.name
+      }] is not a <Route> component. All component children of <Routes> must be a <Route> or <React.Fragment>`
```

Comment:
Do we support Fragments? I didn't think we did. But if so, will they cause this invariant to error out above?

Reply:
Yes, [we do support `React.Fragment`](https://github.com/JamesGelok/react-router/blob/40ccb321c62c4a821f20579de5ad3b714c504e54/packages/react-router/index.tsx#L698-L705). This is what lets you group multiple routes together in a fragment if you prefer to define them that way.

In this PR we would have already returned if the given element is a fragment.